src/Sound.js

```diff
@@ -146,12 +144,14 @@ export default function Sound(props) {
             </div>
             {/* Download stats and Play */}
             <div className="w-6/12 text-right space-x-3 flex justify-end items-center">
-              {/* <Button>  // Removing on Tuesday, November 24, 2020, due to
-                <IonIcon icon={cropSharp} />  // lack of functionality - Hamir
-                <a>Edit</a>
-              </Button> */}
-              {isLoggedIn && loadingState === 1 && (
-                <Dropdown onClick={() => downloadSound(sound)} />
+              {loadingState !== 2 && (
+                <Dropdown
+                  disabled={loadingState !== 1}
+                  onClick={() => {
+                    if (!isLoggedIn) setModalIsOpen(true);
```

Comment:
I initially thought we would do something like
```
                <Dropdown
                  disabled={loadingState !== 1}
                  onClick={() => {
                    if (!isLoggedIn) {
                      <ModalPrompt
                        title="Title of download Modal"
                        onConfirm={navigateToLogin}
                        onClose={closeModal}
                        isOpen={modalIsOpen}
                        onRequestClose={closeModal}
                      />;
                    } else downloadSound(sound);
                  }}
                />
```
if users wanted to download a sound while not logged in. Is an approach like that possible? It didn't work when I tried to implement it.